server/api/impl/upstream/upawslambda/upstream.go

```diff
@@ -27,14 +42,52 @@ func NewUpstream(app *apps.App, aws *aws.Client) *Upstream {
 }
 
 func (u *Upstream) OneWay(call *apps.Call) error {
-	_, err := u.aws.InvokeLambda(u.app.Manifest.AppID, u.app.Manifest.Version, call.URL, lambda.InvocationTypeEvent, call)
+	payload, err := CallToInvocationPayload(call)
+	if err != nil {
+		return errors.Wrap(err, "failed to covert call into invocation payload")
+	}
+
+	_, err = u.aws.InvokeLambda(u.app.Manifest.AppID, u.app.Manifest.Version, u.app.Manifest.Functions[0].Name, lambda.InvocationTypeEvent, payload)
```

Comment:
why do we invoke `u.app.Manifest.Functions[0].Name`?

Reply:
What mechanism do we have to decide which function to invoke?

Reply:
It's in the `call`

Reply:
Maybe I'm misunderstanding. What field in `Call` can be used to to decide which function to invoke? A Call might only have a `Path` which we can only use for routing if a function also have something like a `Path`.